packages/models/src/projects/project_field.rs

```diff
@@ -5,7 +5,7 @@ use dioxus_translate::Translate;
 
 #[derive(Debug, Clone, Eq, PartialEq, Default, ApiModel, Translate, Copy)]
 #[cfg_attr(feature = "server", derive(schemars::JsonSchema, aide::OperationIo))]
-pub enum ProjectArea {
+pub enum ProjectField {
```

Comment:
why do you need to rename this?
now, it can agree, but if you need to rename Type, we must make a separated PR.
These renaming can affect lots of files in UI and API.
So it is important to be quickly reviewed and merged.

But, if it is contained to a PR which modifying lots of code, it will be stuck at reviewing other code.
It may make conflict. 

cc. @biyard/all